submissions/chris-voitova/frogger-game/app.js

```diff
@@ -0,0 +1,138 @@
+const field = {
+  width: 505,
```

Comment:
It's better to move such 'magic numbers' (like 505, 606, 400, 450, 50) into constants with obvious names

Reply:
fixed